x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_gaps/manual_rule_run.cy.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { visitRulesManagementTable } from '../../../../tasks/rules_management';
+import {
+  disableAutoRefresh,
+  manuallyRunFirstRule,
+  manualRuleRunFromDetailsPage,
+} from '../../../../tasks/alerts_detection_rules';
+import {
+  filterByRunType,
+  getExecutionLogTableRow,
+  goToExecutionLogTab,
+  refreshRuleExecutionTable,
+  visitRuleDetailsPage,
+} from '../../../../tasks/rule_details';
+import { getNewRule } from '../../../../objects/rule';
+import { deleteAlertsAndRules } from '../../../../tasks/api_calls/common';
+import { createRule } from '../../../../tasks/api_calls/rules';
+import { login } from '../../../../tasks/login';
+
+describe('Manual rule run', { tags: ['@ess', '@serverless'] }, () => {
```

Comment:
I see that this test needs a FF to be able to pass. Currently FF are not supported on MKI environments, so this test and all the ones that might be affected by it (including API ones), should be skipped from MKI environments.

Reply:
done